subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApi.groovy

```diff
@@ -44,6 +44,7 @@ import java.util.concurrent.TimeUnit
 
 class ToolingApi implements TestRule {
     private static final Logger LOGGER = LoggerFactory.getLogger(ToolingApi)
+    public static final String MINIMUM_SUPPORTED_GRADLE_VERSION = "3.0"
```

Comment:
Is this not the minimum version supported? 
https://github.com/gradle/gradle/blob/68b0581bd7bce9c41b1694f868ccc78280fb35b9/subprojects/tooling-api/src/main/java/org/gradle/tooling/GradleConnector.java#L64

Reply:
I was talking to @donat and accourding to the doc:
https://docs.gradle.org/current/userguide/third_party_integration.html
this seems right. We only mention 3.x+
But I might need to correct it there too.

Reply:
Can you please follow up so that we fix javadoc for 8.2?